es-core/src/components/IList.h

```diff
@@ -302,7 +298,7 @@ class IList : public GuiComponent
 			onScroll(absAmt);
 
 		mCursor = cursor;
-		onCursorChanged((mScrollTier > 0) ? CURSOR_SCROLLING : CURSOR_STOPPED);
+		onCursorChanged((amt != 0) ? CURSOR_SCROLLING : CURSOR_STOPPED);
```

Comment:
Tell us more about this change.

Reply:
I believe the intent of `mScrollTier` was to control and watch the speed of how much the user has scrolled. However mScrollTier is 0 when the user has just scrolled by 1, and only increments to 1 after the user has scrolled by 2. It is a better indicator that check that amt is not 0 to indicate that the user is not scrolling. As this function is also called when the user has stopped scrolling with amt == 0

Reply:
Thanks.